src/types/Person.ts

```diff
@@ -1,4 +1,5 @@
 export interface Person {
+  [x: string]: any;
```

Comment:
fix this linter rule 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/70009205/09770c3b-6dc4-4824-8b06-9f96a64eb021)

Reply:
done